seedlet-blocks/experimental-theme.json

```diff
@@ -1,58 +1,103 @@
 {
 	"global": {
-		"presets": {
-			"font-size": [
-				{
-					"slug": "tiny",
-					"value": "14px"
-				},
-				{
-					"slug": "small",
-					"value": "16px"
-				},
-				{
-					"slug": "medium",
-					"value": "18px"
-				},
-				{
-					"slug": "large",
-					"value": "24px"
-				},
-				{
-					"slug": "extra-large",
-					"value": "28px"
-				},
-				{
-					"slug": "huge",
-					"value": "32px"
-				},
-				{
-					"slug": "gigantic",
-					"value": "48px"
+		"settings": {
+			"color": {
+				"gradients": [
```

Comment:
I noticed that the gradient previews are broken when we supply the gradients via the theme.json:

<img width="268" alt="Screen Shot 2020-10-26 at 2 37 15 PM" src="https://user-images.githubusercontent.com/5375500/97214074-eaff4d80-1798-11eb-9969-18182f7a9939.png">

Other than that, this PR looks good to me.

Reply:
Ouch, this is a bug that needs fixing. It's a bit convoluted and can take a few days of back/forth, so I'd like to offer an alternative to unblock this PR: use raw values to define the gradient presets instead of pointing to the color preset CSS variables.

I'll ping here when the fix lands in Gutenberg.

Reply:
Actually, I've created an [issue](https://github.com/WordPress/gutenberg/issues/26511) to better track this. You can subscribe over there!

Reply:
Thanks @nosolosw! I'll keep an eye on that. @jffng would you mind re-testing? I think this should be all set now.